src/Akka.Streams.Kafka.Tests/RepeatAttribute.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using Xunit;
+using Xunit.Sdk;
+
+namespace Akka.Streams.Kafka.Tests
+{
+    public sealed class RepeatAttribute : DataAttribute
```

Comment:
Helper attribute for Xunit theory, used to repeat a test X numbers of times, useful for testing racy tests